devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -200,14 +201,18 @@ def lowerFanSpeed() {
 }
 
 def low() {
```

Comment:
@dkirker are low/medium/high/max a part of any actual fan capability, or are they hacked in?